Test Runner/UI Tests/FeatureFormViewTests.swift

```diff
@@ -1717,9 +1717,9 @@ final class FeatureFormViewTests: XCTestCase {
         let transformerButton = app.buttons["Transformer, High"]
         
 #if targetEnvironment(macCatalyst)
-        let deleteButton = app.menuItems["delete"]
+        let inlineRemoveAssociationButton = app.menuItems["remove_association"]
 #else
-        let deleteButton = app.buttons["Delete"]
```

Comment:
This change is not directly related to the design spec change. It should've been included in #1288 but I missed it.